pkg/snapshotter/snapshotter_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
I don't think the year of the copyright should be changed here when you are modifying an existing file.  Usually it means starting from this year so it covers 2018 and beyond.

Reply:
Good catch! It was originally a new file but I think git caught it as a move